commit 26ae71da3d43cf1c6f6055c488566908b181d7ac
Author: NAS user <user@nas>
Date:   Mon Feb 24 01:02:29 2014 +0100

    local: kirkwood-dt: Build EHCI as module
    
    Workaround for wrong init order between EHCI and regulator

diff --git a/core/linux-kirkwood-dt/config b/core/linux-kirkwood-dt/config
index 9c02d7a..a60a348 100644
--- a/core/linux-kirkwood-dt/config
+++ b/core/linux-kirkwood-dt/config
@@ -3593,11 +3593,11 @@ CONFIG_USB_WUSB=m
 #
 # CONFIG_USB_C67X00_HCD is not set
 CONFIG_USB_XHCI_HCD=y
-CONFIG_USB_EHCI_HCD=y
+CONFIG_USB_EHCI_HCD=m
 CONFIG_USB_EHCI_ROOT_HUB_TT=y
 CONFIG_USB_EHCI_TT_NEWSCHED=y
-CONFIG_USB_EHCI_PCI=y
-CONFIG_USB_EHCI_HCD_ORION=y
+CONFIG_USB_EHCI_PCI=m
+CONFIG_USB_EHCI_HCD_ORION=m
 # CONFIG_USB_EHCI_HCD_PLATFORM is not set
 # CONFIG_USB_OXU210HP_HCD is not set
 # CONFIG_USB_ISP116X_HCD is not set
